packages/livekit-server-sdk/src/SipClient.ts

```diff
@@ -441,6 +451,9 @@ export class SipClient extends ServiceBase {
       playRingtone: playDialtone,
       playDialtone: playDialtone,
       hidePhoneNumber: hidePhoneNumber,
+      ringingTimeout: new Duration({ seconds: BigInt(ringingTimeout) }),
+      maxCallDuration: new Duration({ seconds: BigInt(maxCallDuration) }),
```

Comment:
when it's not passed in, we should leave these as `undefined` instead of set to 0. this way the server is able to pick sane defaults, versus having different defaults from each client

Reply:
@davidzhao thanks for the review, defaulted the 3 new params to be undefined if not specified